### PR TITLE
Logistic Regression in Keras

### PR DESCRIPTION
**Pull request by MatthewChatham**
In #1 I ran an untuned logistic regression model from `sklearn` that got 92.1% accuracy on the validation set practically out-of-the-box. So, I decided to reimplement the logistic regression in Keras and made it my goal to meet or exceed 92.1% accuracy (which I did). The models are not identical, but are both variants of logistic regression.

Addresses #6 .

### CODE REVIEW

**Comment by MatthewChatham** [comment]
Reviewed, happy with it, closing.
